Gaijinizer patch/Troops.txt

```diff
@@ -6122,40 +6132,44 @@ Mukuro 【Whole book】 missionary position#MTLed
 > BEGIN STRING
 う、うぐっ……！
 > CONTEXT: Troops/126/pages/0/13/Dialogue/kazuya-0
-Uh, aghh......!#MTLed
+Ugh, nggh......!#Ron
 > END STRING
 
 > BEGIN STRING
 \\N[1]は、髪で拘束されてそのまま強引に歩かされている…
 
 その先には、こちらにお尻を向けた躯が待ち構えている…
 > CONTEXT: Troops/126/pages/0/15/Dialogue
-\\N[1] is bound with hair and walks as it is.
-Beyond that, Mukuro with her butt is waiting for you ...#MTLed
+\\N[1] is bound with Mukuro's hair and forced to walk over
+to her. Waiting for him is Mukuro with her rear pointed
+in his direction...#Ron
```

Comment:
Flow and a bit of rearranging...:
>_**\\\N[1] is bound with hair and forced to walk over
as Mukuro waits with her rear pointed at him...**_